### DISCUSSION TITLE
Better file structure organization? "Partials" folder

### DISCUSSION THREAD

**Discussion by brunomdsilva** [filamentphp/filament#11898] [Ideas]
**Problem:**

The current structure of Filament resources can be cluttered, especially for complex resources with many methods, tables, forms, and other components.

**Solution:**

Introduce a Partials folder within the resource folder to organize these components into separate classes. It could also have a command for each of the common resource components

**Benefits:**

- Improved code readability and maintainability.
- Easier to find and modify specific components.
- Promotes code reuse and consistency.

**Example:**

```
└- ModelResource
   └- Pages...
   └- Partials
      └- ModelForm.php
      └- ModelTable.php
      └- ModelInfolist.php
      └- ModelTabs.php
```

**Usage on ModelResource.php**

```php
public static function form(Form $form): Form
{
    return ModelForm::form($form);
}

public static function table(Table $table): Table
{
    return ModelTable::table($table);
}

public static function infolist(Infolist $infolist): Infolist
{
    return ModelInfolist::infolist($infolist);
}
```

**Comment by gldrenthe89** [comment]
+1

Can you elaborato on how you would use the 'Tabs' class?
